setup.py

```diff
@@ -12,17 +12,12 @@
 needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
-if __name__ == "__main__":
-
+try:
```

Comment:
no need to use __name__ == "__main__" as far as I'm aware. We use it MDA core because of all the extra bits we do in the setup file, but I think here it's ok to just stick to what the CMS cookiecutter gave as the first solution.